packages/kubernetes-1.29/kubelet-config

```diff
@@ -91,13 +91,17 @@ kubeReserved:
   {{/if}}
   {{/if}}
   ephemeral-storage: "{{default "1Gi" settings.kubernetes.kube-reserved.ephemeral-storage}}"
+{{#unless settings.kubernetes.reserved-cpus}}
```

Comment:
We should add this for all k8s versions that support this. I believe this is all of ours we currently support - 1.23 through 1.30.